src/constants/URL.ts

```diff
@@ -5,7 +5,11 @@ export const URL = {
 		SIGNUP: '/signup',
 	},
 	USERS: {
-		GET: '/users',
+		SIMPLE: '/users',
```

Comment:
What is SIMPLE? 
![image](https://user-images.githubusercontent.com/44157749/211201435-d1736265-1f8c-4501-a63c-b99a510db495.png)

Reply:
not that simple, it means without params